src/actor/uri.rs

```diff
@@ -4,11 +4,11 @@ use std::{
     sync::Arc,
 };
 
-pub struct ActorPath(Arc<String>);
+pub struct ActorPath(Arc<str>);
```

Comment:
shouldn't this be `&str`?

Reply:
This will be way harder to do as you need to add a lifetime parameter and it should be &'a str.